content/en/admin/optional/object-storage.md

```diff
@@ -192,6 +192,13 @@ During batch delete operations, S3 providers may perodically fail or
 timeout while processing deletion requests. Mastodon will back off and
 retry the request up to this maximum number of times.
 
+### `S3_KEY_PREFIX`
+
+Default: none
```

Comment:
Default line should be after explanation of value.

Reply:
Are you certain? All the other environment variables in this files have the Default before the explanation.

https://github.com/mastodon/documentation/blob/3beb730c57b4a3650f8a0ea861a70dfd9c265200/content/en/admin/optional/object-storage.md?plain=1#L110-L148

Reply:
Ah this page appears to be the opposite of the main config option page. We should probably standardize this.

Reply:
probably better to fix this in a separate PR.

Reply:
#1465